test/integration/stop/test_stop.py

```diff
@@ -4,10 +4,9 @@
 import pytest
 import ttapp
 
-from utils import (get_tarantool_version, pid_file, run_command_and_get_output,
-                   run_path, wait_file)
+import utils
```

Comment:
Why do we need this change?

Reply:
In my opinion it's more convenient, because once you need any function from utils you just use it, no need to add the corresponding item into `from utils import ...` line (I see no problem to write `utils.<some_func>` instead of just `<some_func>`, moreover for me the former even looks better because it explicitly says that this function is external). Also after modifying `from util import ...` you would need to make sure it passes flaky:isort verification.
Besides I'm using this approach (I mean `import utils`) in the similar `test_*.py` modules that was added (not modified) in this PR.

Reply:
Ok, fine.